src/content/docs/ai-monitoring/compatibility-requirements-ai-monitoring.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Compatibility and requirements for AI monitoring
+metaDescription: 'Compatibility and requirements for AI monitoring'
+freshnessValidatedDate: never
+---
+
+AI monitoring allows agents to recognize and capture AI data. AI monitoring has different library compatibility requirements depending on what language you used for your AI-powered app. 
+
+## Compatible AI libraries [#compatibility] 
+
+Below you can check if you're using the correct agent version for AI monitoring. You can also confirm whether our APM agents can collect data from the libraries or frameworks you use in your AI-powered app.
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Agent version
+            </th>
+            <th>
+                Supported libraries
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+            [Go version 3.31.0 and above](/docs/apm/agents/go-agent/get-started/go-agent-compatibility-requirements/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [Go OpenAI library](https://github.com/sashabaranov/go-openai) versions 3.4.0 and above
+            * [AWS SDK for Go v2](https://github.com/aws/aws-sdk-go-v2) versions 1.6.0 and above
+            </td>
+        </tr>
+        <tr>
+            <td>
+            [Node.js version 11.13.0 and above](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [OpenAI Node.js API library](https://www.npmjs.com/package/openai/v/4.0.0-beta.4) versions 4.0.0 and above. If your model uses streaming, the Node.js agent supports versions 4.12.2 and above
+            * [AWS SDK for JavaScript BedrockRuntime Client](https://www.npmjs.com/package/@aws-sdk/client-bedrock-runtime) versions 3.474.0 and above
+            * [LangChain.js](https://www.npmjs.com/package/langchain/v/0.1.17) versions 0.1.17 and above
+            </td>
+        </tr>
+        <tr>
+            <td>
+            [Python version 9.9.0 and above](/docs/apm/agents/python-agent/getting-started/compatibility-requirements-python-agent/#digital-intelligence-platform)
+            </td>
+            <td>
+            * [OpenAI](https://pypi.org/project/openai/) library versions 1.13.3 and above
+            * [Boto3 AWS SDK for Python](https://docs.aws.amazon.com/pythonsdk/) 
```

Comment:
There technically is a minimum version that supports Bedrock. Do we want to include it here? I see for Node we do call out a minimum version.